apps/web/package.json

```diff
@@ -19,11 +19,11 @@
     "check-changed-files": "ts-node scripts/ts-check-changed-files.ts"
   },
   "engines": {
-    "node": "16.x || 18.x",
+    "node": "16.x || 18.x || 19.x",
```

Comment:
Also thinking supporting 19.x requires testing our dependencies for 19.x support and I'm predicting that not all libraries do..

Reply:
🤷 

```bash
error @boxyhq/saml-jackson@1.8.1: The engine "node" is incompatible with this module. Expected version ">=14.18.1 <=18.x". Got "19.7.0"
```